ocean_lib/ocean/ocean_assets.py

```diff
@@ -96,6 +124,14 @@ def _process_service_descriptors(
         if access_service_descriptor:
             _service_descriptors.append(access_service_descriptor)
         if compute_service_descriptor:
+            if trusted_algorithms:
```

Comment:
move the if too;
the function name add_trusted_algorithms is ambiguous. It does not ADD anything, only if you move the part that modifies the service descriptor to that function. Otherwise it formats publisher trusted algorithms, and should be named format_publisher_trusted_algorithms and moved into a utils file (no need for "self" at all). I think that is the best course of action is the second one, to avoid name clashes with the utilitary function.